truenas/webui
# NAS-114420 / 22.02 / Cloud Sync Task UI does not keep Directory/Files checkbox checked

To test:
Data Protection -> Cloud Sync Task -> Add or Edit -> set `Direction` to PUSH -> check `Directory/Files` checkbox -> Save
Confirm to remained checked when edit again.
